### DISCUSSION TITLE
Fundamentos do SQL SERVER - Azure Data Studio

### DISCUSSION THREAD

**Discussion by dosanjosg** [balta-io/forum#490] [Tira Dúvidas] [answered]
Na aula de ferramentas, para criar uma new connection dentro do azure quando seleciono a opção "database", as opções que deveriam aparecer não aparecem e surge o seguinte erro na tela: 
![Captura de tela 2024-02-12 192657](https://github.com/balta-io/forum/assets/135039362/e75d4f4e-a0fb-4c58-ba64-de110c0ced26)

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado -> https://github.com/balta-io/forum/discussions/388
